src/main/java/lotto/domain/LottoGames.java

```diff
@@ -37,12 +37,11 @@ public int getGameCount() {
         return lottoGames;
     }
 
-    public HitResults result(WinNumbers winNumbers) {
+    public HitResults start(WinNumbers winNumbers) {
         HitResults hitResults = new HitResults(payMoney);
         for (LottoGame lottoGame : lottoGames) {
             hitResults.add(lottoGame.lotto(winNumbers));
         }
```

Comment:
이렇게도 표현 가능이 가능 할 것같아요! 
```suggestion
        lottoGames.stream().map(lottoGame -> lottoGame.lotto(winNumbers)).forEach(hitResults::add);
```

makerNumbers()메서드 안에 반복문도 
이런식으로 표현이 가능하겠네요 :) ->  lottoGames.forEach(LottoGame::makeNumbers);